aspnetcore/mvc/views/overview.md

```diff
@@ -1,228 +1,332 @@
-keywords: ASP.NET Core,
+description: Learn how views handle the app's data presentation and user interaction in ASP.NET Core MVC.
+keywords: ASP.NET Core,view,MVC,razor,viewmodel,viewdata,viewbag
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 09/26/2017
 ms.topic: article
 ms.assetid: 668c320d-c050-45e3-8161-2f460dc93b2f
 ms.technology: aspnet
 ms.prod: asp.net-core
 uid: mvc/views/overview
 ---
-# Rendering HTML with views in ASP.NET Core MVC
+# Views in ASP.NET Core MVC
 
-By [Steve Smith](https://ardalis.com/)
+By [Steve Smith](https://ardalis.com/) and [Luke Latham](https://github.com/guardrex)
 
-ASP.NET Core MVC controllers can return formatted results using *views*.
+In the **M**odel-**V**iew-**C**ontroller (MVC) pattern, the *view* handles the app's data presentation and user interaction. A view is an HTML template with embedded [Razor markup](xref:mvc/views/razor). Razor markup is code that interacts with HTML markup to produce a webpage that's sent to the client.
 
-## What are Views?
+In ASP.NET Core MVC, views are *.cshtml* files that use the [C# programming language](/dotnet/csharp/) in Razor markup. Usually, view files are grouped into folders named for each of the app's [controllers](xref:mvc/controllers/actions). The folders are stored in a in a *Views* folder at the root of the app:
 
-In the Model-View-Controller (MVC) pattern, the *view* encapsulates the presentation details of the user's interaction with the app. Views are HTML templates with embedded code that generate content to send to the client. Views use [Razor syntax](razor.md), which allows code to interact with HTML with minimal code or ceremony.
+![Views folder in Solution Explorer of Visual Studio is open with the Home folder open to show About.cshtml, Contact.cshtml, and Index.cshtml files](overview/_static/views_solution_explorer.png)
 
-ASP.NET Core MVC views are *.cshtml* files stored by default in a *Views* folder within the application. Typically, each controller will have its own folder, in which are views for specific controller actions.
+The *Home* controller is represented by a *Home* folder inside the *Views* folder. The *Home* folder contains the views for the *About*, *Contact*, and *Index* (homepage) webpages. When a user requests one of these three webpages, controller actions in the *Home* controller determine which of the three views is used to build and return a webpage to the user.
 
-![Views folder in Solution Explorer](overview/_static/views_solution_explorer.png)
+In addition to views that are matched to controller actions, you can use [partial views](xref:mvc/views/partial) for reusable parts of webpages. For example, a partial view is useful for a login panel that appears in several views. You can also use [layouts](xref:mvc/views/layout) to reduce the repetition of common user interface elements. A layout can include the app's header with navigation elements and the app's footer for use with several views.
```

Comment:
Is this example of a login panel what we really want to use? We recommend using a View Component for this over [here](https://docs.microsoft.com/en-us/aspnet/core/mvc/views/view-components#introducing-view-components). This could cause confusion.

Reply:
The MVC template is using a partial.
```cshtml
@await Html.PartialAsync("_LoginPartial")
```
However, it's probably best to not use that as an example. I'll put in a touch-up there and use a different example or just cut that bit out.

Reply:
Also, I don't like the way partials, layouts, and view components are mentioned. The text is uneven when it describes other view features. It just needs a small touch here and there ... I'll get it on this next commit.